### DISCUSSION TITLE
ch1-basic/ch1-01-genesis

### DISCUSSION THREAD

**Discussion by giscus** [chai2010/advanced-go-programming-book#645] [General]
# ch1-basic/ch1-01-genesis

Go语言高级编程

https://chai2010.cn/advanced-go-programming-book/ch1-basic/ch1-01-genesis.html

**Comment by wjc77** [comment]
闽南理工wjc已学习完

**Comment by mlee-msl** [comment]
golang并发执行多任务，并聚合收集多任务执行结果与执行状态： https://github.com/mlee-msl/taskgroup/tree/main
